modules/deploy-models-bias-monitoring.adoc

```diff
@@ -0,0 +1,52 @@
+:_module-type: PROCEDURE
+
+[id="deploy-models-bias-monitoring_{context}"]
+= Deploying models
+
+. Navigate to the model-namespace created in the setup section: 
++
+[source]
+----
+oc project model-namespace
+----
+
+. Deploy the model's storage container:
++
+[source]
+----
+oc apply -f 2-BiasMonitoring/resources/model_storage_container.yaml
+----
+
+. Deploy the OVMS 1.x serving runtime: 
++
+[source]
+----
+oc apply -f 2-BiasMonitoring/resources/ovms-1.x.yaml
+----
+
+. Deploy the first model: 
++
+[source]
+----
+oc apply -f 2-BiasMonitoring/resources/model_alpha.yaml
+----
+
+. Deploy the second model: 
++
+[source]
+----
+oc apply -f 2-BiasMonitoring/resources/model_beta.yaml
+----
+
```

Comment:
you could make the following a Verification section - but that is really a future enhancement